src/patient-registration/section/section-wrapper.component.tsx

```diff
@@ -8,7 +8,7 @@ export const SectionWrapper = ({ id, children, name, index }) => {
   return (
     <div id={id} style={{ marginTop: '2rem' }}>
       <h3 className={styles.productiveHeading02} style={{ color: '#161616' }}>
-        {index + 1}. {name}
+        {index + 1}. {t(name)}
```

Comment:
I would expect the `name` prop to be given in translated form.

This is quite a substantial / elementary decision:

1. Either not being consistent / not knowing what's happening inside
2. Never do something inside; always expect the props to be handled as it should be

Personally, I prefer (2) a lot. You never have to think about if this component does translation or not, you can always decide if you want to have a translated string etc. -> so please remove `t(name)` and translate *before* passing in to `SectionWrapper`.

Reply:
Yeah Sure. My bad. Sorry for changing the internal structure. I'll look a way around for it asap.

Reply:
I made the text translation when the sections are being fetched from config. It is the part from where we fetch the sections inside the patient-registration-component, and removed the translation from section-wrapper-component. I hope it would be correct this time.